scripts/scil_compute_bundle_specific_fodf_metrics.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Script to compute bundle-specific fODF metrics, derived for fitting a
```

Comment:
bundle-specific is not the proper name here I think. To have such a name, the input to the script would require a bundle, in my opinion.

suggestion:
scil_compute_bingham_fodf_metrics.py

Reply:
Done in ba1e733 :+1: